go/sign/access_key_singer_test.go

```diff
@@ -0,0 +1,39 @@
+package sign
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+)
+
+func ExampleAccessKeyHTTPSigner_Sign() {
```

Comment:
Without being aware of the requirements, just wanted to mention changing ExampleAccessKeyHTTPSigner_Sign to TestAccessKeyHTTPSigner_Sign() (and import "testing") will help in using 'go test' command for unit testing.

Reply:
> access_key_si**ng**er_test.go should probably be renamed to access_key_si**gn**er_test.go

Good catch. I'll make the change